### ISSUE TITLE
Modified U+1D9FF SIGNWRITING HEAD glyphs lack anchors

### ISSUE THREAD

**Issue by dscorbett** [googlefonts/noto-fonts#2302]
### Font
NotoSansSignWriting-Regular.otf

### Where the font came from, and when
Site: https://github.com/googlefonts/noto-fonts/blob/12ea769bd0bf283691d3da92089d2e9018c9d700/unhinted/otf/NotoSansSignWriting/NotoSansSignWriting-Regular.otf
Date: 2022-03-21

### Font version
Version 2.003

### Issue
Modified forms of U+1D9FF SIGNWRITING HEAD lack anchors, so subsequent marks fail to combine with them. As in other Noto fonts, unanchored marks overlap the _following_ glyph for some strange reason, which means that instead of looking merely suboptimal it is completely wrong.

### Character data
𝧿𝪡𝨀
U+1D9FF SIGNWRITING HEAD
U+1DAA1 SIGNWRITING ROTATION MODIFIER-2
U+1DA00 SIGNWRITING HEAD RIM

### Screenshot
![𝧿𝪡𝨀](https://user-images.githubusercontent.com/1124347/159361849-d02e7ae8-148f-4ed2-b943-9db4bc46ad0d.png)

**Comment by marekjez86** [comment]
@dscorbett : is this a fontmake issue?

**Comment by dscorbett** [comment]
NotoSansSignwriting.glyphs includes anchor points for `u1D9FF_F2` which are not present in the final font file, so I suppose it could be a fontmake issue.

**Comment by simoncozens** [comment]
Yes, it is. Because of the naming convention `<something>_<something>`, glyphsLib assumes that `u1D9FF_F2` is a GDEF class 2 (ligature) glyph and so it looks for mark-to-ligature anchors `_<anchorname>_1`, `_<anchorname>_2` instead of `_<anchorname>`. I've complained about this assumption before (https://github.com/googlefonts/glyphsLib/issues/756) but the only suggested fixes make the problem worse.

We can work around it by manually assigning all these `<something>_<something>` to be Category=Letter, Subcategory=Other in the Glyphs sources.
